docs/dev-guide/contributing.md

```diff
@@ -3,14 +3,18 @@ id: dev-guide-contributing
 title: Contributing Guidelines
 ---
 
-# How to Contribute 🤝
-We greatly appreciate your willingness to contribute. Before you start working however, please take a moment to read and follow this brief guide.
+# 🤝 How to Contribute 
+We greatly appreciate your willingness to contribute ❤️ Before you start working however, please take a moment to read and follow this brief guide.
 
-# Reporting Issues and Asking Questions 💭
+# 📥 Reporting Issues and Asking Questions 
 
-We prefer issues to be discussed first in the [community forum](https://community.jitsi.org/) and when confirmed, then an issue can be opened in the [issue tracker](https://github.com/jitsi/jitsi-meet/issues).
+- We prefer issues to be discussed first in the [community forum](https://community.jitsi.org/) and when confirmed, then an issue can be opened in the [issue tracker](https://github.com/jitsi/jitsi-meet/issues).
```

Comment:
Shall we re-discuss this @damencho ? I think it's not a very natural flow. People find the issue tracker first. Then we send them away, but the issue might come back once confirmed.

That creates some friction.

I think improviing the issue template / questionaire would help, and once an issue is identified as something that should be treated in the forum we can close it as such. Thoughts?

Reply:
I suggest moving forward with this PR as it's an improvement and describes the current procedure. Let's re-discuss separately.

Reply:
Good point. I'll approve as I have no other comments.